docs/index.md

```diff
@@ -1,3 +1,30 @@
 # Introduction
 
-TDB...
+Saleor SDK Python is a Python library which purpose is to contain code that is repeated in different Saleor applications.
+
+You should read [Saleor's Documentation](https://docs.saleor.io/docs/3.x/developer/extending/overview) on the topic to get a grasp on the concepts of the "app framework" Saleor comes with.
+
+This SDK is agnostic of any framework, which makes it possible to use in all sorts of web (or otherwise) frameworks and even scripts.
+
+## What about Python App Framework?
+
+The old [saleor-app-framework-python](https://github.com/mirumee/saleor-app-framework-python) was very opinionated on how an application should be crated. It required FastAPI and in a specific version of it.  
+
+## Installation
+
+```
+pip install saleor-sdk-python
+```
+
+## Key features
```

Comment:
I would consider dropping this part and moving 2-3 most important things from this to "introduction".

Reply:
Decided to keep it here.
However when we will introduce docs for marina then we will decide to move it to `introduction`